### PR TITLE
WIP-data-received-success

### PR DESCRIPTION
**Pull request by Rancha124**
Data is received successfully but I couldn't use that data to render dynamically

### CODE REVIEW

File: src/Components/AddStudent.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  };
+
+  handleChangeStateName = (event) => {
+    this.setState({ stateName: event.target.value });
+  };
+
+
+  handleSubmit = (e) => {
+e.preventDefault();
+  history.push("./show-data");
```
Comment (yaswanthkapollo) [inline src/Components/AddStudent.js]: This should be changed to history.push("/show-data");

File: src/Components/AddStudent.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    this.setState({ cityName: event.target.value });
+  };
+
+  handleChangeStateName = (event) => {
+    this.setState({ stateName: event.target.value });
+  };
+
+
+  handleSubmit = (e) => {
+e.preventDefault();
```
Comment (yaswanthkapollo) [inline src/Components/AddStudent.js]: Have a payload and call axios.post with the payload

File: src/Components/AddStudent.js [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+                  placeholder="State Name"
+                  onChange={handleChange}
+                  onBlur={handleBlur}
+                  value={values.stateName}
+                  className={
+                    touched.stateName && errors.stateName ? "error" : null
+                  }
+                >
+                  <option></option>
+                  <option>Andhra Pradhesh</option>
```
Comment (yaswanthkapollo) [inline src/Components/AddStudent.js]: Keep options in a array and iterate it

File: src/Components/EditData.js [hunk 4]
```diff
@@ -0,0 +1,386 @@
+import React from "react";
```
Comment (yaswanthkoneri) [inline src/Components/EditData.js:1]: Most of the EditData component is duplicated, so try to reuse the AddStudent component for edit as well

File: src/Components/FullDetails.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+import { Table, Button } from "react-bootstrap";
+import history from "../../src/services/history";
+class FullDetails extends React.Component {
+  state = {
+    student: [],
+    dummy: false,
+  };
+  componentDidMount() {
+    const id = this.props.match.params.id;
+    axios.get(`http://localhost:3001/students/${id}`).then((result) => {
```
Comment (yaswanthkoneri) [inline src/Components/FullDetails.js]: Keep all the calls in service file and use it

**Pull request merged**